src/app/components/register-card/register-card.component.ts

```diff
@@ -50,22 +61,23 @@ export class RegisterCardComponent implements OnInit {
 
   onRegisterUser(): void {
     const user: User = {
-      name: this.registerForm.get('registerNameControl')!.value.replace(/\s+/g, ' ').trim(), 
-      email: this.registerForm.get('registerEmailControl')!.value.trim(), 
-      password: this.registerForm.get('registerPasswordControl')!.value
+      name: this.registerForm.get('registerNameControl')!.value,
+      email: this.registerForm.get('registerEmailControl')!.value,
+      password: this.registerForm.get('registerPasswordControl')!.value,
     };
 
     this.registerService.register(user).subscribe({
       next: (data: User) => {
-        this.responseUserObject = {...data};
-        this.snackBar.open(`User ${this.responseUserObject.name} has been registered.`, '', {
-          duration: 2500
-        });
+        this.responseUserObject = { ...data };
+        this.snackBar.open(
+          `User ${this.responseUserObject.name} has been registered.`,
+          'Close'
```

Comment:
This should have been left with a set duration. If not, the snackbar will stick there until dismissed by the user.